File: WePeiYang/LostFound/Model/LostFoundItem.m
//
//  LostFoundItem.m
//  WePeiYang
//
//  Created by Qin Yubo on 16/2/14.
//  Copyright © 2016年 Qin Yubo. All rights reserved.
//

#import "LostFoundItem.h"

@implementation LostFoundItem

+ (NSDictionary *)mj_replacedKeyFromPropertyName {
    return @{@"index": @"id",
             @"name": @"name",
             @"title": @"title",
             @"place": @"place",
             @"time": @"time",
             @"phone": @"phone",
             @"lostType": @"lost_type",
             @"otherTag": @"other_tag",
             @"foundPic": @"found_pic"};
}

@end

File: WePeiYang/LostFound/Model/LostFoundItem.swift
//
//  LostFoundItem.swift
//  WePeiYang
//
//  Created by Qin Yubo on 16/4/22.
//  Copyright © 2016年 Qin Yubo. All rights reserved.
//

import UIKit
import ObjectMapper

class LostFoundItem: NSObject, Mappable {
    
    var index = ""
    var name = ""
    var title = ""
    var place = ""
    var time = ""
    var phone = ""
    var lostType = 0
    var otherTag = ""
    var foundPic = ""
    
    required init?(_ map: Map) {
        
    }
    
    func mapping(map: Map) {
        index <- map["id"]
        name <- map["name"]
        title <- map["title"]
        place <- map["place"]
        time <- map["time"]
        phone <- map["phone"]
        lostType <- map["lost_type"]
        otherTag <- map["other_tag"]
        foundPic <- map["found_pic"]
    }
}
